### ISSUE TITLE
False errors

### ISSUE THREAD

**Issue by morf** [Respect/Validation#8]
```
try {
    $location = v::arr()

        ->key('timezone', v::string()->length(99, 100), false)
            ->setName('timezone')

        ->key('state', v::string(), false)
            ->setName('state')

        ->key('city', v::string(), false)
            ->setName('city');

    $location->assert($testData);

} catch (\Exception $e) {
   var_dump($e->findMessages('timezone', 'state', 'city'));
}
```

if

```
$testData = array('timezone' => 'fail', 'city' => 'ok');
```

or

```
$testData = array('timezone' => 'fail');
```

Result:

```
array(3) {
  ["timezone"]=>
  string(38) "Key timezone must be valid on timezone"
  ["state"]=>
  string(0) ""
  ["city"]=>
  string(41) "These {{failed}} rules must pass for city"
}
```

(always the last)
But if

```
$testData = array('city' => 'ok');
```

no throw exception
